lib/ui/views/homepage/widgets/product_item.dart

```diff
@@ -33,12 +31,11 @@ class ProductItem extends StatelessWidget {
             Stack(
               alignment: AlignmentDirectional.topEnd,
               children: [
-                Image.asset(
-                  recipe?.dishImageUrl ?? '--',
+                Image.network(
+                  recipe?.dishImageUrl ?? 'https://media.istockphoto.com/id/1404501005/photo/jollof-rice.jpg?s=1024x1024&w=is&k=20&c=DTns3r_STc4HRBMouVl8IgHeuWhnBdz8X4DRkbDxWjQ=',
```

Comment:
This is actually not the solution to our issue. I told them in the last class what needs to be done and someone should create a PR for it. Men, it's well.

Reply:
Ohh I missed this. I no no wetin u tell them. I just saw the null error and made it work